app/views/handlers/content.py

```diff
@@ -36,11 +35,4 @@ def _render_block(self, block_id):
     def _get_page_title(self, transformed_block):
         content = transformed_block.get("content")
         if content:
-            if isinstance(content["title"], str):
-                content_title = content["title"]
-            else:
-                content_title = content["title"]["text"]
-
-            page_title = f'{content_title} - {self._schema.json["title"]}'
-
-            return safe_content(page_title)
+            return self._get_safe_page_title(content["title"])
```

Comment:
Summary uses this so I think we still need to check content

Reply:
This is in the content handler, I can't see where it is used by the summary.

Reply:
Block type 'Summary' still comes through here, which doesn't have a content element. When I change it to your suggest it just errors on any summary
```
return self._get_safe_page_title(transformed_block ["content"]["title"])
KeyError: 'content'
```
Its not something I have changed, so can't say whether its right that the summary block is using this

Reply:
Ah, I see it now. I don't think it's right that the `Summary` handler extends the `Content` handler, I'll raise a card for it thought rather than extending this pull request further.